Sources/MeiliSearchCore/PackageVersion.swift

```diff
@@ -1,18 +1,15 @@
 import Foundation
-#if canImport(FoundationNetworking)
-  import FoundationNetworking
-#endif
 
-internal struct PackageVersion {
+public struct PackageVersion {
   /// This is the current version of the meilisearch-swift package
-  private static let current = "0.15.0"
+  public static let current = "0.15.0"
```

Comment:
minor: let's make sure we update [the Podspec](https://github.com/meilisearch/meilisearch-swift/blob/main/MeiliSearch.podspec) where this line of code is referenced (the path to this file)